---
author: Lindsey Becker
categories:
- Barley
- QTL
- mycobiome
date: "2023-07-02"
draft: false
excerpt: The genetic bases of how Barley's genetic variation impacts the leaf mycobiome in the context of disease remain largely unknown. Here, we are using a mapping population of Barley Scald resistant and susceptible genotypes and applying metabarcoding strategies to identify interactions between host and fungal taxa that may influence disease outcomes.
featured: true
layout: single
subtitle: ""
tags:
- Barley
- QTL
- mycobiome
- Scald
title: Mapping microbial taxa and host disease severity
---

### This is a project that I was invited to collaborate on with Patrick Hayes (Barley breeder at Oregon State) and Posy Busby. We are using Quantitative trait locus (QTL) mapping to identify barley genomic regions associated with variations in fungal taxa.

------------------------------------------------------------------------

Malting barley is a critical component of fermented beverages produced globally. Barley scald, a fungal disease with global presence that can cause up to 30% yield loss for growers. While minor host resistance genes have been identified, much less is known about how fungal communities are correlated with resistance to barley scald.

Here, we are characterizing the barley leaf fungal communities and performing quantitative trait locus (QTL) analysis to identify host genetic loci associated with abundant fungal taxa. We are integrating microbiome QTL (mbQTL) to uncover microbiomeâ€“disease associations and identify candidate leaf fungal taxa associated with barley scald resistance.

![Microbiome QTL](mbQTL.png)

------------------------------------------------------------------------
