---
author: Lindsey Becker
categories:
- Barley
- QTL
- microbiome
date: "2023-07-02"
draft: false
excerpt: The molecular bases of how Barley's genetic variation impacts the leaf microbiome in the context of disease remain largely unknown. Here, we are using a mapping population of Barley Scald resistant and susceptible genotypes and applying metabarcoding strategies to identify interactions between host and bacterial taxa that may influence disease outcomes.
featured: true
layout: single
subtitle: ""
tags:
- Barley
- QTL
- microbiome
- Scald
title: Mapping microbial taxa and host disease severity
---

### This is a new project that I was invited to collaborate in with the Patrick Hayes (Barley breeder at OSU) and Posy Busby. We are using Quantitative trait locus (QTL) mapping to identify barley genomic regions associated with variations in bacterial taxa.

------------------------------------------------------------------------

Malting barley is a critical component of fermented beverages produced globally. Barley scald, a fungal pathogen with a global presence can cause up to 30% yield loss for growers. While minor host resistance genes have been identified, much less is known about how microbial communities are correlated with resistance to barley scald.

Here, we are characterizing the barley leaf microbiome and performing quantitative trait locus (QTL) analysis to identify host genetic loci associated with abundant bacterial taxa. We are integrating microbiome QTL (mbQTL) to uncover microbiomeâ€“disease associations and identify candidate leaf bacterial taxa associated with barley scald resistance.

![Microbiome QTL](mbQTL.png)

------------------------------------------------------------------------
